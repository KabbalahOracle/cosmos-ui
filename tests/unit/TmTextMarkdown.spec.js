import { shallowMount } from "@vue/test-utils"
import htmlBeautify from "html-beautify"
import { TmTextMarkdown } from "@/index"

describe("TmTextMarkdown.vue", () => {
  let wrapper
  beforeEach(async () => {
    wrapper = shallowMount(TmTextMarkdown)
  })

  it("has the expected html structure", () => {
    expect(htmlBeautify(wrapper.html())).toMatchSnapshot()
  })
})
