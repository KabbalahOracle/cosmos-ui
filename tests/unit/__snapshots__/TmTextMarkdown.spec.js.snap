// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TmTextMarkdown.vue has the expected html structure 1`] = `
<div class="tm-text-markdown">
  <h2 id="loading%E2%80%A6">
    Loading…
  </h2>
  <p>
    Loading…
  </p>
</div>
`;
